packages/rpc-graphql/src/loaders/transaction.ts

```diff
@@ -20,14 +20,7 @@ async function loadTransaction(
     rpc: Rpc<GetTransactionApi>,
     { signature, ...config }: TransactionLoaderArgs,
 ): Promise<TransactionLoaderValue> {
-    // @ts-expect-error FIX ME: https://github.com/solana-labs/solana-web3.js/pull/2052
-    return await rpc
-        .getTransaction(
-            signature,
-            // @ts-expect-error FIX ME: https://github.com/solana-labs/solana-web3.js/pull/2052
```

Comment:
I think that the real FIXME is https://github.com/microsoft/TypeScript/issues/43187

> Generic inference to an overloaded function always chooses the last signature.

Reply:
Yup. Smh.

Reply:
#2133